cogs/misc.py

```diff
@@ -44,6 +86,36 @@ async def _8ball(self, ctx, *, question=None):
             random.choices(tuple(outcomes.keys()), tuple(outcomes.values()))[0]
         )
 
+    @commands.command(name="proton", aliases=["protondb"])
+    async def proton(self, ctx):
+        """Retrieve ProtonDB stats for Project Winter."""
+
```

Comment:
Just add a quick `None` check for `self.proton_pw` so that if it's None we don't get weird errors please.